lib/ukrand/chacha.c

```diff
@@ -126,23 +126,45 @@ void uk_swrand_init_r(struct uk_swrand *r, unsigned int seedc,
 	__u32 k[8], iv[2];
 
 	for (i = 0; i < 8; i++)
-		k[i] = _infvec_val(10, seedv, i);
+		k[i] = infvec_val(10, seedv, i);
 
-	iv[0] = _infvec_val(seedc, seedv, i);
-	iv[1] = _infvec_val(seedc, seedv, i + 1);
+	iv[0] = infvec_val(seedc, seedv, i);
+	iv[1] = infvec_val(seedc, seedv, i + 1);
 
-	_uk_key_setup(r, k);
-	_uk_iv_setup(r, iv);
+	uk_key_setup(r, k);
+	uk_iv_setup(r, iv);
 
 	r->k = 16;
 }
 
-__u32 uk_swrand_randr_r(struct uk_swrand *r)
+static int uk_chacha_init(struct uk_init_ctx *ictx __unused)
+{
+	unsigned int seedc = 10;
```

Comment:
I don't remember what the `10` was for and I can't find the comment in this PR. But can you leave a comment here maybe explaining this value? Ditto for line below